RELEASE/scripts/sl_ascend.ash

```diff
@@ -12737,15 +12737,15 @@ boolean L9_chasmBuild()
 		buffMaintain($effect[Carol of the Hells], 50, 1, 1);
 		buffMaintain($effect[Song of Sauce], 150, 1, 1);
 
-		if(get_property("sl_aggressiveBlechHouse").to_boolean())
+		if(get_property("sl_beta_test").to_boolean())
```

Comment:
Eh, I'd rather this be separate from beta test. I still think the better behavior is stopping when we encounter blech house and aren't expecting it, personally, since that means we might actually get reports of the issue instead of it going unnoticed.

Reply:
So add back in "sl_aggressiveBlechHouse" or make it always stop regardless of mode? I'd vote for the former personally?

Reply:
Eh I guess I'm fine with either of those, although I'd kind of prefer to avoid cluttering the preferences. But it's probably worth having a preference in this case, I guess?